src/test/groovy/graphql/schema/idl/SchemaGeneratorTest.groovy

```diff
@@ -1634,13 +1633,14 @@ class SchemaGeneratorTest extends Specification {
         argInt.getDirective("thirdDirective") != null
 
         def intDirective = argInt.getDirective("intDirective")
-        intDirective.name == "intDirective"
-        intDirective.arguments.size() == 1
-        def directiveArg = intDirective.getArgument("inception")
+        def intAppliedDirective = argInt.getAppliedDirective("intDirective")
```

Comment:
Key change: the applied directive contains the value, and the directive contains the default value